HomeWork_06/HW_06_Task_02/StringModifier.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace HW_06_Task_02
+{
+    public class StringModifier
+    {
+        StringBuilder builder = new StringBuilder();
+        private string[] array;
```

Comment:
for private fields the best practice to add under score `_` as a first character:

`private string[] _array;`
`string _str, _shortestWord, _longestWord = String.Empty;`